loader_hub/file/image/base.py

```diff
@@ -19,78 +19,96 @@ class ImageReader(BaseReader):
 
     """
 
-    def __init__(self) -> None:
+    def __init__(self, text_type: str = "plain_text") -> None:
         """Init parser."""
-        try:
-            import torch  # noqa: F401
-        except ImportError:
-            raise ValueError("install pytorch to use the model")
-        try:
-            from transformers import DonutProcessor, VisionEncoderDecoderModel
-        except ImportError:
-            raise ValueError("transformers is required for using DONUT model.")
-        try:
-            import sentencepiece  # noqa: F401
-        except ImportError:
-            raise ValueError("sentencepiece is required for using DONUT model.")
-        try:
-            from PIL import Image  # noqa: F401
-        except ImportError:
-            raise ValueError(
-                "PIL is required to read image files." "Please run `pip install Pillow`"
+
+        if text_type == "plain_text":
+            try:
```

Comment:
This isn't something you introduced, but we don't need to do try/except statements here anymore because requirements.txt is installed prior to this module. You should add `pytesseract` to the requirements.txt file and then remove the try/excepts.

Reply:
Got it. Updated.